application/models.py

```diff
@@ -1,3 +1,115 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+# defines model for a single step (or box) from prescribing guide flowchart
+class Step(models.Model):
+    # Fields
+    name = models.CharField(help_text='Enter step title', max_length=25)
+    description = models.TextField(help_text='Enter step description')
+    
+    # Metadata
+    class Meta: 
+        ordering = ['name']
+
+    # Methods
+    def get_absolute_url(self):
+        """Returns the url to access a particular instance of MyModelName."""
+        return reverse('model-detail-view', args=[str(self.id)])
```

Comment:
I don't see reverse being defined or imported anywhere, does this work?

Reply:
Yeah this is just how it returns like an order of the instance. Apparently it doesn't work right if it's not reversed. I just found it on the Django tutorial site.